lms/static/js/student_account/components/PasswordResetConfirmation.jsx

```diff
@@ -0,0 +1,140 @@
+/* globals gettext */
+
+import 'whatwg-fetch';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Button, StatusAlert } from '@edx/paragon/static';
+
+import PasswordResetInput from './PasswordResetInput';
+
+// NOTE: Use static paragon with this because some internal classes (StatusAlert at least)
+// conflict with some standard LMS ones ('alert' at least). This means that you need to do
+// something like the following on any templates that use this class:
+//
+// <link type='text/css' rel='stylesheet' href='${STATIC_URL}paragon/static/paragon.min.css'>
+//
+
+class PasswordResetConfirmation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password1: '',
+      password2: '',
+      passwordsMatch: true,
+      isValid: true,
+      validationMessage: '',
+    };
+    this.onBlurPassword1 = this.onBlurPassword1.bind(this);
+    this.onBlurPassword2 = this.onBlurPassword2.bind(this);
+  }
+
+  onBlurPassword1(password) {
+    this.updatePasswordState(password, this.state.password2);
+    this.validatePassword(password);
+  }
+
+  onBlurPassword2(password) {
+    this.updatePasswordState(this.state.password1, password);
+  }
+
+  updatePasswordState(password1, password2) {
+    this.setState({
+      password1,
+      password2,
+      passwordsMatch: !password1 || !password2 || (password1 === password2),
```

Comment:
Does this need to be `and`ed like

```javascript
!password && !password2 && (password1 === password2)
```

Reply:
Gotcha! Makes sense...

If that's the case, and this is probably a nitpick, but it seems that the name `passwordsMatch` is inaccurate? Maybe `showPasswordConfirmationError` is more accurate?

Reply:
You're not wrong. And since you tripped on it, might as well fix it for the next person.